test/gtest/ucp/test_ucp_am.cc

```diff
@@ -2050,6 +2065,34 @@ class test_ucp_am_nbx_rndv_ppln : public test_ucp_am_nbx_rndv {
                                     << stats_node->cls->counter_names[cntr];
     }
 
+    bool is_ppln_supported(ucs_memory_type_t mem_type)
+    {
+        const auto ep     = sender().ep();
+        const auto config = ucp_ep_config(ep);
+
+        for (auto i = 0; i < config->key.num_lanes; ++i) {
+            const auto lane = config->key.rma_bw_lanes[i];
+            if (lane == UCP_NULL_LANE) {
+                break;
+            }
+            if (ucp_ep_md_attr(ep, lane)->reg_mem_types & UCS_BIT(mem_type)) {
+                const auto iface_attr =
+                    ucp_worker_iface_get_attr(sender().worker(),
+                                              ucp_ep_get_rsc_index(ep, lane));
+                if (iface_attr->cap.flags & UCT_IFACE_FLAG_PUT_ZCOPY) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
```

Comment:
why is it needed?
there is `ucp_test_base::entity::is_rndv_put_ppln_supported()`

Reply:
Thanks for spotting this, this is not needed, and not actually used, it's just a leftover after rebasing on master